[Zero Trust] Improved explainer on seat manageament (#5575)
cloudflare/cloudflare-docs: content/cloudflare-one/identity/users/seat-management.md

# Manage user seats

Cloudflare Zero Trust subscriptions consist of seats that active users in your account consume. Active users are added to the Zero Trust dashboard through any authentication event. 

> The amount of user seats available in your Zero Trust account depends on the amount of users you purchase. If you want to increase the number of seats available, you will have to purchase more users. Learn more about adding and removing users from your account in the [Zero Trust FAQ](/cloudflare-one/faq/teams-getting-started-faq/#how-do-i-change-my-subscription-plan).

Reviewer: ```suggestion
The amount of user seats available in your Zero Trust account depends on the amount of users you purchase. If you want to increase the number of seats available, you will have to purchase more users. Learn more about adding and removing seats from your account in the [Zero Trust FAQ](/cloudflare-one/faq/teams-getting-started-faq/#how-do-i-change-my-subscription-plan).
```

A user who authenticates will hold their seat until you [remove the user](#remove-a-user) from your account. By default, inactive users will not be [automatically removed](#remove-inactive-users) from your account. You can remove a single user or all users at any time, and those users will immediately stop counting against your subscription.

## Revoke vs remove a user

When you revoke a user, this action will terminate active sessions, but will not remove the user’s consumption of an active seat. On the other hand, when you remove a user, this action will revoke their session and remove that user registry, freeing up one seat from your account. 

## Check number of Active Users

You can check for the number of active users in the [Zero Trust dashboard](https://dash.teams.cloudflare.com) home. 

## Remove a user

1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), navigate to **My Team** > **Users**.
2. Search for and select the user you want to remove.
3. Click **Update Status** > **Remove users**.

If a user is removed, and then authenticates once more, they will count as a seat again.

## Remove inactive users

> Cloudflare Zero Trust can automatically remove any user who does not log in to an Access application or who does not trigger a device enrollment event within a specified time period (between 2 months and 1 year). These users will no longer count against your seat count.

Reviewer: ```suggestion
Cloudflare Zero Trust can automatically remove any user who does not trigger an [authentication event](#what-constitutes-an-authentication-event) within a specified time period (between 2 months and 1 year). These users will no longer count against your seat count.
```
